xpdacq/xpdacq.py

```diff
@@ -244,6 +244,29 @@ def calibration(sample, scanplan, **kwargs):
     prun(sample, _scanplan)
     # this way is cleaner and dark is collected as well. but "no calibration file" warning might appear while people are doing calibration run.
 
+def background(sample, scanplan, **kwargs):
```

Comment:
I create a copy of `scanplan` inside background function called `_scanplan` so that we can play with md without changing original `scanplan` object.